tests/integration/test_log.py

```diff
@@ -85,6 +88,21 @@ def test_log_rollback(cluster):
     log.read()
     assert match(r'.*INCRBY.*333', str(log.entries[-1].data()))
 
+    # make sure after double resume, crc chain has been maintained
+    # before second resume get # entries
+    node1_log_size = cluster.node(1).info()['raft_log_entries']
+
+    # kill all nodes (1 first, as we don't want any additional log entries)
+    cluster.node(1).terminate()
+    cluster.node(2).terminate()
+    cluster.node(3).terminate()
+
+    # restart 1, won't have an active cluster, but should have loaded log
+    cluster.node(1).start()
+    # need to give it time to read log
+    time.sleep(1)
```

Comment:
It can be flaky on a slow build, maybe we can do `cluster.node(1).wait_for_election()`

Reply:
by definition in this case, there wont be one (other 2 nodes are down) that's why I did it this way.  I was trying to figure out a different way to do it (i.e. wait on something that says node is up)

Reply:
How about `cluster.node(1).wait_for_info_param('raft_state', 'up')`

Reply:
done